randovania/games/dread/json_data/Ghavoran.txt

```diff
@@ -831,14 +833,13 @@ Extra - asset_id: collision_camera_012
       All of the following:
           Lay Bomb or Lay Power Bomb
           Any of the following:
-              Space Jump
+              Gravity Suit or Space Jump
               Flash Shift and Wall Jump (Beginner)
               All of the following:
                   Grapple Beam
                   Flash Shift or Spider Magnet or Grapple Movement (Beginner) or Use Spin Boost
-              Any of the following:
-                  # This room allows for a bomb jump out of the water without a WBJ
-                  Gravity Suit or Lay Bomb
+              Water Bomb Jump (Beginner) and Lay Cross Bomb
+              Water Bomb Jump (Intermediate) and Lay Normal Bomb
```

Comment:
This should also probably include IBJ (Intermediate) since you do still need to IBJ.

Reply:
You reach directly up with one cross bomb, and for normal bomb there's a walljump here. It's possible to make the walljump both from a water bomb jump and from an out of water bomb jump that is technically not a water bomb jump. Though, it appears to me that the former is still a little bit more precise than the full underwater version, which is why I left it at intermediate.

Reply:
It would just be consistent with other uses of verticality with bombs, where even if a single cross bomb is enough it's labeled as IBJ. Could see it where WBJ may imply IBJ in cases though.